IntegrationExample/Processing/View/ProcessingViewModel.swift

```diff
@@ -31,67 +32,71 @@ final class ProcessingViewModel: ObservableObject {
     private var cancellables = Set<AnyCancellable>()
     
     private var session: MimiProcessingSession? {
-        core.processing.session.value
+        core.processing.session
     }
-    
-    init(core: MimiCore,
-         headphoneConnectivity: PartnerHeadphoneConnectivityController) {
+
+    // MARK: - Init
+
+    init(core: MimiCore = .shared, headphoneConnectivity: PartnerHeadphoneConnectivityController) {
         self.core = core
-        self.headphoneConnectivity = headphoneConnectivity
-        
-        self.isSessionAvailable = core.processing.session.value != nil
-        self.isEnabled = core.processing.session.value?.isEnabled.value ?? false
-        self.intensity = core.processing.session.value?.intensity.value ?? 0
-        self.presetId = core.processing.session.value?.preset.value?.id
-        
+        self.isSessionAvailable = core.processing.session != nil
+
         self.isHeadphoneConnected = headphoneConnectivity.state.value != .disconnected
-        
+        self.headphoneConnectivity = headphoneConnectivity
+
         self.isUserLoggedIn = core.auth.currentUser != nil
-        
         core.auth.observable.addObserver(self)
-        
-        subscribeToProcessingSession()
+
         subscribeToHeadphoneConnectivityState()
+        subscribeToProcessingSession()
     }
-    
+
+    // MARK: - Headpone connectivity
+
+    func simulateHeadphoneConnection(isConnected: Bool) {
+        headphoneConnectivity.simulateHeadphoneConnection(isConnected: isConnected)
+    }
+
+    // MARK: - Subscribers
+
     private func subscribeToProcessingSession() {
-        core.processing.session
+        core.processing.sessionPublisher
+            .receive(on: DispatchQueue.main)
             .sink { [weak self] session in
                 guard let session else {
                     self?.isSessionAvailable = false
                     return
                 }
-                // Since the Slider sends updates continously, setting the delivery mode to discreet
-                // allows us to debounce the updates.
-                session.intensity.deliveryMode = .discrete(seconds: 0.2)
```

Comment:
As we don't have the `deliveryMode` anymore, I think it's good to set the `uiControlDebounceBehavior` in the configuration.